torchao/quantization/quant_api.py

```diff
@@ -393,7 +393,7 @@ def insert_subclass(lin):
     return insert_subclass
 
 
-def int4_weight_only(group_size=128, layout_type=TensorCoreTiledLayoutType(inner_k_tiles=8)):
+def int4_weight_only(group_size=128, layout_type=TensorCoreTiledLayoutType(inner_k_tiles=8), use_hqq=False):
```

Comment:
for this one I'm planning to have a separate hqq function that can work with all dtypes actually

Reply:
thats fine but the way it was setup to work made no sense before so this is a strict improvement.